lib/Transformation.ts

```diff
@@ -110,7 +110,6 @@ export function transformLiteral(lit: RDF.Literal): E.Literal<any> {
 
     case DT.XSD_INTEGER:
     case DT.XSD_DECIMAL:
-
```

Comment:
Could you undo the changes in this file? That will make the diff a bit cleaner :-)